test/test_google/test_google_drive.py

```diff
@@ -0,0 +1,28 @@
+import unittest
+import os
+from parsons import GoogleDrive
+
+# Test Slides: https://docs.google.com/presentation/d/19I-kicyaJV53KoPNwt77KJL10fHzWFdZ_c2mW4XJaxc
+
+
+@unittest.skipIf(
+    not os.environ.get("LIVE_TEST"), "Skipping because not running live test"
+)
+class TestGoogleDrive(unittest.TestCase):
+    def setUp(self):
+
+        self.gd = GoogleDrive()
+
+    def test_get_permissions(self):
+
+        file_id = "19I-kicyaJV53KoPNwt77KJL10fHzWFdZ_c2mW4XJaxc"
+        p = self.gd.get_permissinos(file_id)
```

Comment:
This looks like a typo - and a sign that the tests aren't getting run, because this should have raised an error.